<!DOCTYPE html>
<html lang="en">

<head>
	<meta charset="utf-8"/>
	<title>Teschendorff lab / papers</title>
	<meta name="viewport" content="width=device-width, initial-scale=1.0">
	
    <!-- Customized Bootstrap + Font Awesome + Solarized -->
    <link href="/css/style.css" rel="stylesheet" media="screen">

	<!-- Favicon -->
	<link rel="shortcut icon" href="icon.jpg"/>	
			
	<!-- Google Analytics -->	
	<script>
		(function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
		(i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
		m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
		})(window,document,'script','//www.google-analytics.com/analytics.js','ga');

		ga('create', 'UA-58618307-1', 'teschendorff-lab.github.io');
		ga('send', 'pageview');

	</script>
			
</head>

<body>
	
	<div id="header">
		<nav class="navbar navbar-default navbar-static-top" role="navigation">	 
			<div class="container">	
				
				<div class="navbar-header">
					<object class="pull-left logo" data="icon.jpg"></object>
					<a class="navbar-brand" href="/">Teschendorff Lab</a>
				</div>
				
				<ul class="nav navbar-nav navbar-right">
                    <!-- 
                    <li>
                    
                    <a href="/blog/">blog</a>
                    </li> -->
					
					<li class="active">
					
					<a href="/papers/">Papers</a></li>
					
					<li>
					
					<a href="/projects/">Projects</a></li>	  
					
					<li>
					
					<a href="/team/">People</a></li>	  	  
				</ul>
			</div>
		</nav>	
	</div>
	
	<div class="container">	
		

<div class="bigspacer"></div>	
<div class="row">
<div class="col-md-2"></div>
<div class="col-md-10">
	<div class="media">
	</div>
		<div class="media-body">
			<h3>Papers in <strong>bold</strong> were led or co-led by members of the Teschendorff lab.</h3>
		</div>
</div>
</div>
<hr class="featurette-divider">

    
    

    <div class="row">
		<div class="col-md-2 paperbox">
				<div class="media-body">
				
				<div class="head media-heading" id="2018-ref" class="off">2018</div>
				
				</div>
		</div>
		<div class="col-md-10 paperbox">
<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.nature.com/articles/s41592-018-0213-x" class="off">Identification of differentially methylated cell types in epigenome-wide association studies</a></strong></div><p class="note">Zheng SC, Breeze CE, Beck S, Teschendorff AE. 2018 Nov 30</p><p class="note"><i>Nat Methods</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://doi.org/10.1038/s41563-018-0241-z" class="off">Avoiding common pitfalls in machine learning omic data science</a></strong></div><p class="note">Teschendorff AE. 2018 Nov 26</p><p class="note"><i>Nat Materials</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://doi.org/10.18632/aging.101666" class="off">Cell and tissue type independent age-associated DNA methylation changes are not rare but common</a></strong></div><p class="note">Zhu T, Zheng SC, Paul DS, Horvath S, Teschendorff AE. 2018</p><p class="note"><i>Aging (Albany NY)</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.ncbi.nlm.nih.gov/pubmed/30304472" class="off">Detection of epigenetic field defects using a weighted epigenetic distance-based method</a></strong></div><p class="note">Wang Y, Qian M, Ruan P, Teschendorff AE, Wang S. 2018 Oct 10</p><p class="note"><i>Nucleic Acids Res</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://academic.oup.com/bib/advance-article/doi/10.1093/bib/bby093/5115275" class="off">Quantifying Waddington's epigenetic landscape: a comparison of single-cell potency measures</a></strong></div><p class="note">Shi J, Teschendorff AE, Chen W, Chen L, Li T. 2018 Oct 5</p><p class="note"><i>Brief Bioinform</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.aging-us.com/article/101533/text" class="off">Epigenetic clocks galore: a new improved clock predicts age-acceleration in Hutchinson Gilford Progeria Syndrome patients</a></strong></div><p class="note">Teschendorff AE. 2018 Aug 21</p><p class="note"><i>Aging (Albany  NY)</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.futuremedicine.com/doi/10.2217/epi-2018-0037" class="off">A novel cell-type deconvolution algorithm reveals substantial contamination by immune cells in saliva, buccal and cervix</a></strong></div><p class="note">Zheng SC, Webster AP, Dong D, Feber A, Graham DG, Sullivan R, Jevons S, Lovat LB, Beck S, Widschwendter M, Teschendorff AE. 2018 Jul</p><p class="note"><i>Epigenomics</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC5994057" class="off">Tensorial blind source separation for improved analysis of multi-omic data</a></strong></div><p class="note">Teschendorff AE, Jing H, Paul DS, Virta J, Nordhausen K. 2018 Jun 8</p><p class="note"><i>Genome Biol</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.sciencedirect.com/science/article/pii/S2352396418301531?via%3Dihub" class="off">DNA Methylation Patterns in Normal Tissue Correlate more Strongly with Breast Cancer Status than Copy-Number Variants</a></strong></div><p class="note">Gao Y, Widschwendter M, Teschendorff AE. 2018 May</p><p class="note"><i>EBioMedicine</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.nature.com/articles/nrg.2017.86" class="off">Statistical and integrative system-level analysis of DNA methylation data</a></strong></div><p class="note">Teschendorff AE, Relton CL. 2018 Mar</p><p class="note"><i>Nat Rev Genet</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://academic.oup.com/bioinformatics/article/34/3/398/4265460" class="off">Tumor Origin Detection with Tissue-Specific miRNA and DNA methylation Markers</a></strong></div><p class="note">Tang W, Wan S, Yang Z, Teschendorff AE, Zou Q. 2018 Feb 1</p><p class="note"><i>Bioinformatics</i></p></div></div></div></div><div class="bigspacer"></div>

</div><!--------------2017--------------->
					<div class="col-md-2 paperbox">
				<div class="media-body">
				
				<div class="head media-heading" id="2017-ref" class="off">2017</div>
				
				</div>
		</div>
			<div class="col-md-10 paperbox">
<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-017-1366-0" class="off">Systems-epigenomics inference of transcription factor activity implicates aryl-hydrocarbon-receptor inactivation as a key event in lung cancer development</a></strong></div><p class="note">Chen Y, Widschwendter M, Teschendorff AE. 2017 Dec 20</p><p class="note"><i>Genome Biol</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="" class="off">The potential of circulating tumor DNA methylation analysis for the early detection and management of ovarian cancer</a></strong></div><p class="note">Widschwendter M, Zikan M, Wahl B, Lempiäinen H, Paprotka T, Evans I, Jones A, Ghazali S, Reisel D, Eichner J, Rujan T, Yang Z, Teschendorff AE, Ryan A, Cibula D, Menon U, Wittenberger T. 2017 Dec 22</p><p class="note"><i>Genome Med</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://bmjopen.bmj.com/content/7/10/e016708" class="off">Are objective measures of physical capability related to accelerated epigenetic age? Findings from a British birth cohort</a></strong></div><p class="note">Simpkin AJ, Cooper R, Howe LD, Relton CL, Davey Smith G, Teschendorff A, Widschwendter M, Wong A, Kuh D, Hardy R. 2017 Nov 1</p><p class="note"><i>BMJ Open</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://onlinelibrary.wiley.com/doi/abs/10.1002/ijc.31014" class="off">Roadmap for investigating epigenome deregulation and environmental origins of cancer</a></strong></div><p class="note">Herceg Z, Ghantous A, Wild CP, Sklias A, Casati L, Duthie SJ, Fry R, Issa JP, Kellermayer R, Koturbash I, Kondo Y, Lepeule J, Lima SCS, Marsit CJ, Rakyan V, Saffery R, Taylor JA, Teschendorff AE, Ushijima T, Vineis P, Walker CL, Waterland RA, Wiemels J, Ambatipudi S, Degli Esposti D, Hernandez-Vargas H. 2018 Mar 1</p><p class="note"><i>Int J Cancer</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.futuremedicine.com/doi/abs/10.2217/epi-2016-0153" class="off">Cell-type deconvolution in epigenome-wide association studies: a review and recommendations</a></strong></div><p class="note">Teschendorff AE, Zheng SC. 2017 May</p><p class="note"><i>Epigenomics</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://academic.oup.com/cardiovascres/article/113/6/692/3743778" class="off">DNA methylome analysis reveals distinct epigenetic patterns of ascending aortic dissection and bicuspid aortic valve</a></strong></div><p class="note">Pan S, Lai H, Shen Y, Breeze C, Beck S, Hong T, Wang C, Teschendorff AE. 2017 May 1</p><p class="note"><i>Cardiovasc Res</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.impactjournals.com/oncotarget/index.php?journal=oncotarget&page=article&op=view&path[]=14013&pubmed-linkout=1" class="off">Aberrant regulation of RANKL/OPG in women at high risk of developing breast cancer</a></strong></div><p class="note">Kiechl S, Schramek D, Widschwendter M, Fourkala EO, Zaikin A, Jones A, Jaeger B, Rack B, Janni W, Scholz C, Willeit J, Weger S, Mayr A, Teschendorff A, Rosenthal A, Fraser L, Philpott S, Dubeau L, Keshtgar M, Roylance R, Jacobs IJ, Menon U, Schett G, Penninger JM. 2017 Jan 17</p><p class="note"><i>Oncotarget</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://doi.org/10.1093/bioinformatics/btx513" class="off">Bioinformatics</a></strong></div><p class="note">Yuan Tian, Tiffany J Morris, Amy P Webster, Zhen Yang, Stephan Beck, Andrew Feber, Andrew E Teschendorff; ChAMP: Updated Methylation Analysis Pipeline for Illumina BeadChips. 
https://doi.org/10.1093/bioinformatics/btx513

</p><p class="note"><i>2017</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://www.nature.com/articles/ncomms15599" class="off">Single-cell entropy for accurate estimation of differentiation potency from a cell's transcriptome</a></strong></div><p class="note">Teschendorff AE, Enver T. 2017 Jun 1</p><p class="note"><i>Nat Commun</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.nature.com/nmeth/journal/v14/n3/full/nmeth.4187.html" class="off">Correcting for cell-type heterogeneity in epigenome-wide association studies: revisiting previous analyses</a></strong></div><p class="note">Zheng SC, Beck S, Jaffe AE, Koestler DC, Hansen KD, Houseman AE, Irizarry RA, Teschendorff AE. 2017 Feb 28</p><p class="note"><i>Nat Methods</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-017-1511-5" class="off">A comparison of reference-based algorithms for correcting cell-type heterogeneity in Epigenome-Wide Association Studies</a></strong></div><p class="note">Teschendorff AE, Breeze CE, Zheng SC, Beck S. 2017 Feb 13</p><p class="note"><i>BMC Bioinformatics</i></p></div></div></div></div><div class="bigspacer"></div>
	    </div><!-----------------------------2016---------------------------->
<div class="col-md-2 paperbox">
	<div class="media-body">
		<div class="head media-heading" id="2016-ref" class="off">2016</div>
				
	</div>
</div>
			
<div class="col-md-10 paperbox">
	
<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.nature.com/articles/ncomms13555?WT.ec_id=NCOMMS-20161130&spMailingID=52882237&spUserID=ODkwMTM2NjQyNgS2&spJobID=1048363436&spReportId=MTA0ODM2MzQzNgS2" class="off">Increased DNA methylation variability in type 1 diabetes across three immune effector cell types</a></strong></div><p class="note">Paul DS, Teschendorff AE, Dang MA, Lowe R, Hawa MI, Ecker S, Beyan H,Cunningham S, Fouts AR, Ramelius A, Burden F, Farrow S, Rowlston S, Rehnstrom K,Frontini M, Downes K, Busche S, Cheung WA, Ge B, Simon MM, Bujold D, Kwan T, Bourque G, Datta A, Lowy E, Clarke L, Flicek P, Libertini E, Heath S, Gut M,Gut IG, Ouwehand WH, Pastinen T, Soranzo N, Hofer SE, Karges B, Meissner T, Boehm BO, Cilio C, Elding Larsson H, Lernmark Å, Steck AK, Rakyan VK, Beck S, Leslie RD. 2016 Nov 29</p><p class="note"><i>Nat Commun</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.sciencedirect.com/science/article/pii/S2211124716314796" class="off">eFORGE: A Tool for Identifying Cell Type-Specific Signal in Epigenomic Data</a></strong></div><p class="note">Breeze CE, Paul DS, van Dongen J, Butcher LM, Ambrose JC, Barrett JE, Lowe R, Rakyan VK, Iotchkova V, Frontini M, Downes K, Ouwehand WH, Laperle J, Jacques PÉ, Bourque G, Bergmann AK, Siebert R, Vellenga E, Saeed S, Matarese F, Martens JH, Stunnenberg HG, Teschendorff AE, Herrero J, Birney E, Dunham I, Beck S. 2016 Nov 15</p><p class="note"><i>Cell Rep</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1064-3" class="off">Correlation of an epigenetic mitotic clock with cancer risk</a></strong></div><p class="note">Yang Z, Wong A, Kuh D, Paul DS, Rakyan VK, Leslie RD, Zheng SC, Widschwendter M, Beck S, Teschendorff AE. 2016 Oct 3</p><p class="note"><i>Genome Biol</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://nar.oxfordjournals.org/content/early/2016/09/14/nar.gkw802.long" class="off">Teschendorff, Bernd Denecke, Qiong Lin, Martin Widschwendter, Elmar Weinhold, Ivan G</a></strong></div><p class="note">Marie Kalwa, Sonja Hänzelmann, Sabrina Otto, Chao-Chung Kuo, Julia Franzen, Sylvia Joussen, Eduardo Fernandez-Rebollo, Björn Rath, Carmen Koch, Andrea Hofmann, Shih-Han Lee, Andrew E. 2016 : gkw802v1-gkw802</p><p class="note"><i>Costa, and Wolfgang Wagner The lncRNA HOTAIR impacts on mesenchymal stem cells via triple helix formation Nucleic Acids Res</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://genomemedicine.biomedcentral.com/articles/10.1186/s13073-016-0342-8" class="off">The multi-omic landscape of transcription factor inactivation in cancer</a></strong></div><p class="note">Teschendorff AE, Zheng SC, Feber A, Yang Z, Beck S, Widschwendter M. 2016 Aug 25</p><p class="note"><i>Genome Med</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.pnas.org/content/early/2016/07/20/1604558113.abstract" class="off">Menopause accelerates biological aging</a></strong></div><p class="note">Morgan E.Levine, Ake T.Lu, Brian H.Chen, Dena G.Hernandez, Andrew B.Singleton, Luigi Ferrucci, Stefania Bandinelli, Elias Salfati, JoAnn E.Manson, Austin Quach, Cynthia D.J.Kusters, Diana Kuh, Andrew Wong, Andrew E.Teschendorff, Martin Widschwendter, Beate R.Ritz, Devin Absher, Themistocles L.Assimes, and Steve Horvath. doi: 10.1073/pnas.1604558113</p><p class="note"><i>PNAS</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://journals.lww.com/ijgc/pages/articleviewer.aspx?year=9000&issue=00000&article=98687&type=abstract" class="off">DNA Methylation Signatures in Vaginal Fluid Samples for Detection of Cervical and Endometrial Cancer</a></strong></div><p class="note">Doufekas K*, Zheng SC*, Ghazali S, Wong M, Mohamed Y, Jones A, Reisel D, Mould T, Olaitan A, Macdonald N, Teschendorff AE, Widschwendter M. 2016 Jun 2</p><p class="note"><i>Int J Gynecol Cancer</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.ncbi.nlm.nih.gov/pubmed/27104983#" class="off">Epigenetic drift, epigenetic clocks and cancer risk</a></strong></div><p class="note">Zheng SC, Widschwendter M, Teschendorff AE. 2016 Apr 22</p><p class="note"><i>Epigenomics</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://bmcbioinformatics.biomedcentral.com/articles/10.1186/s12859-016-1056-z" class="off">Stochastic epigenetic outliers can define field defects in cancer</a></strong></div><p class="note">Teschendorff AE, Jones A, Widschwendter M. 2016 Apr 22</p><p class="note"><i>BMC Bioinformatics</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://epigeneticsandchromatin.biomedcentral.com/articles/10.1186/s13072-016-0058-4" class="off">Tissue-independent and tissue-specific patterns of DNA methylation alteration in cancer</a></strong></div><p class="note">Chen Y, Breeze CE, Zhen S, Beck S, Teschendorff AE. 2016 Mar 8</p><p class="note"><i>Epigenetics Chromatin</i></p></div></div></div></div><div class="bigspacer"></div>

<div class="row"><div class="col-md-2 paperbox"><div class="media-body"></div></div><div class="col-md-10 paperbox"><div class="media"><div class="media-body"><div class="smallhead media-heading"><strong><a href="http://www.nature.com/ncomms/2016/160129/ncomms10478/full/ncomms10478.html" class="off">DNA methylation outliers in normal breast tissue identify field defects that are enriched in cancer</a></strong></div><p class="note">Andrew E Teschendorff, Yang Gao, Allison Jones, Matthias Ruebner, Matthias W.Beckmann, David L.Wachter, Peter A.Fasching & Martin Widschwendter. 2016</p><p class="note"><i>Nature Communications</i></p></div></div></div></div><div class="bigspacer"></div>





<div class="bigspacer"></div>

	
	</div>
		
	<div id="footer"><span style="display:none">foo</span></div>
	    		
	<!-- MathJax support -->
	<script type="text/javascript"
	 src="http://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script>
	<!-- MathJax config -->
	<script src="/js/mathjax.config.js"></script>	
			
</body>
</html>

